archunit-junit/junit5/engine/build.gradle

```diff
@@ -1,23 +1,29 @@
 plugins {
     id 'archunit.java-release-conventions'
+    id 'java-test-fixtures'
 }
 
 ext.moduleName = 'com.tngtech.archunit.junit5.engine'
 
-ext.minimumJavaVersion = JavaVersion.VERSION_1_8
+ext.minimumJavaVersion = JavaVersion.VERSION_1_9
```

Comment:
We can't do this or we will break `archunit-junit5` support for all users out there that use Java 8!

Reply:
Ummm...

The getting started doc for ArchUnit says: 

> ArchUnit can be obtained from Maven Central.
> 
> Gradle
> dependencies {
>    testImplementation 'com.tngtech.archunit:archunit:0.23.1'
> }

and the `archunit` module already declares `minimumJavaVersion` to be 9, so I was under the impression ArchUnit already dropped support for Java 8

Reply:
All previous releases (i.e. up to 0.23.1) actually support _Java 7;_ its support has only recently been dropped with #833. 😉

`minimumJavaVersion` is only set to `JavaVersion.VERSION_1_9` (or `JavaVersion.VERSION_16`) for individual Gradle tasks `compileJdk9mainJava`, `compileJdk9testJava`, `jdk9Test` (or `compileJdk16testJava`, `jdk16Test`, respectively).

Reply:
Duly noted. 

I've downgraded the version back to Java 8 (turns out Java 9 features are not yet used in this PR), but **please** consider adding this to the contributing guide as it's pretty confusing, especially given that some modules use `ext.minimumJavaVersion = JavaVersion.VERSION_1_9`